doc/getting-started.md

```diff
@@ -33,12 +33,12 @@ Let's deconstruct the above URL.
 - Scheme: HTTPS
 - Host/Address: httpbin.org
 - Port: 443
-- URI: get
+- URI: /get
 - Query Parameters: foo1, foo2
 
 ## Prerequisites
 
-- The guide uses docker and docker-compose to setup APISIX. But if you have already installed APISIX via other methods, then you can skip to step 2.
+- This guide use docker and docker-compose to setup APISIX. But if you have already installed APISIX in other ways, you can just skip to step 2.
```

Comment:
`uses`

Reply:
Could you please add the link for [step 2](XXX)?

Reply:
fixed